src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,40 @@
+package duke;
```

Comment:
I like how the project is organised! Every class is in a package, and the Tasks are in their own subpackage as well.